# EvolutionaryAssemblyFromProteinDomains
Evolutionary assembly of the plant terrestrialization toolkit from protein domains
* We want to trace the evolutionary predisposition the green lineage had at the time of terrestrialization.
* Terrestrialization is a complete change in environment from water, and these acted like new or intensified stress factors on the green lineage.
* Thus, to track the evolutionary predisposition which led to successfully overcoming the challenges of a new environment we carefully annotate, dissect and analyse the stress proteome of the extant Green lineage.

—Key Concepts—
-Already existing Concept-
1. Orthology
-New/Redefined Concept-
1. Embryophytic Domain: Protein domain which has an Embryophytic ancestor, that is, present in at least one Bryophyte species and at least one Tracheophyte species
2. Latent Genetic Potential: Key functional protein domains having Last Common Ancestor prior (LCA) to the LCA of protein with corresponding function

Data: (Figure 1 A)
* We take 6 species from Cholorophyte algae lineage, 7 from Streptophyte algae lineage, 5 from Bryophytes lineage and 14 Tracheophytes. 5 species from Cyanobacteria were taken as an outgroup lineage.

Stress-annotation (Figure 1B)
* First 57,796 orthogroups were obtained from 37 proteome. This is obtained by using OrthoFinder2.
* Next we need to annotate stress-related proteins across our dataset. Following are existing ways of gene and protein ontology:
    * Extensive experiments have been done on A.thaliana and P.patens in the green lineage. Thus, we stress-annotated 37 proteomes using TAIR10 and PEATmoss respectively. From this method, we stress-annotated 4902 orthogroups
    * Eggnogmapper is a known tool with an extensive database used for gene ontology purposes. We stress-annotated 17349 orthogroups.
* In order to avoid experimental bias and tool-based randomness we overlap the two methods to have our final stress-annotation. We finally filtered 2475 stress-annotated orthogroups from 57,796 orthogroups.

Overview of Stress-annotation (Figure 1C, 1D)
* To have an overview of stress-annotation, overlaps of stress-annotated protein domains, proteins and orthogroups across our dataset lineages and various response to stresses are shown.

Distribution of Stress-annotation (Figure 2A)
* Using the overlap annotation approach explained earlier, the stress-annotated orthogroups and corresponding protein domains are seen here. A standard pattern is observed in both, that is, the unique number of stress orthogroups and stress protein domains increase from Cyanobacteria to Tracheophytes. The number of proteins and average number of protein domains also increases similarly.
